app/main/actions/search.js

```diff
@@ -36,15 +36,24 @@ const DEFAULT_SCOPE = {
  * @param {Function} callback Callback function that receives used search term and found results
  */
 const eachPlugin = (term, display) => {
+  const externalSettings = config.get('external') || {}
   // TODO: order results by frequency?
   Object.keys(plugins).forEach(name => {
+    const settings = externalSettings[name]
+    if (settings) {
+      Object.keys(settings).forEach(key => {
+        settings[key] = settings[key].value
```

Comment:
@BrainMaestro no, I mean why do you need nested object with `value => '...'`? Now `externalSettings` is something like:

```js
{
  pluginName: {
    language: {
      value: 'en'
    },
    token: {
      value: 'user-token'
    }
  }
}
```

But it could be simplified as:

```
{
  pluginName: {
    language: 'en',
    token: 'user-token'
  }
}
```